Snowball-Finance/axial-frontend
# rewards page design modification

To test:
1. Please test the design for rewards page as per figma
2. Please go through all the functionality of rewards page.
3. See list of rewards pool tokens data is correct.
4. Check the pool showed paused when required with deposit button disabled.
5. Check the deposit functionality is working fine
6. Check the withdraw is working fine
7. Please check the validation while depositing and withdrawing.

Needs to implement:
1. Claim rewards
2. One know issue while depositing on as4d pool, informed @mohsenParvar (will fixed it together), other pools deposit and withdraw is working fine.
3. Usd balance on My Share components showing wrong data.
